cmd/logcli/main.go

```diff
@@ -109,9 +109,9 @@ func newQueryClient(app *kingpin.Application) *client.Client {
 		return nil
 	}
 
-	app.Flag("addr", "Server address.").Default("https://logs-us-west1.grafana.net").Envar("GRAFANA_ADDR").Action(addressAction).StringVar(&client.Address)
-	app.Flag("username", "Username for HTTP basic auth.").Default("").Envar("GRAFANA_USERNAME").StringVar(&client.Username)
-	app.Flag("password", "Password for HTTP basic auth.").Default("").Envar("GRAFANA_PASSWORD").StringVar(&client.Password)
+	app.Flag("addr", "Server address.").Default("http://localhost:3100").Envar("LOKI_ADDR").Action(addressAction).StringVar(&client.Address)
+	app.Flag("username", "Username for HTTP basic auth.").Default("").Envar("LOKI_USERNAME").StringVar(&client.Username)
```

Comment:
It would be nice to document this in the help strings as well, eg. `Username for HTTP basic auth. Uses LOKI_USERNAME from environment, if set.`

I didn't know until now that logcli already supports this.

Reply:
I think that's a fair, @sandlis can you update the flag description ?

Reply:
Added it. Please check.
Thanks!